.github/docker/rez-win-base/Dockerfile

```diff
@@ -5,6 +5,8 @@
 ARG WINDOWS_VERSION
 FROM mcr.microsoft.com/windows/servercore:$WINDOWS_VERSION
 
+LABEL org.opencontainers.image.description="WARNING: This is an internal image and should not be used outside of the rez repository!"
```

Comment:
This label gets rendered in the GitHub packages UI. I'm also considering renaming the images to `rez-internal...` to make it really clear that these images are not for public consumption in any way.